Products/ZenUtils/controlplane/client.py

```diff
@@ -141,14 +142,31 @@ def getChangesSince(self, age):
         :param age: How far back to look, in milliseconds, for changes.
         """
         query = {"since": age}
-        response = self._dorequest("/services", query=query)
+        response = self._dorequest(self._servicesEndpoint, query=query)
         body = ''.join(response.readlines())
         response.close()
         decoded = ServiceJsonDecoder().decode(body)
         if decoded is None:
             decoded = []
         return decoded
 
+    def updateServiceProperty(self, service, prop):
+        """
+        Updates the launch property of a service.
```

Comment:
Minor thing, if you make any other change, this Updates *any* property of a service, not just the launch property.